receiver/otlpreceiver/factory_test.go

```diff
@@ -148,13 +148,13 @@ func TestCreateMetricReceiver(t *testing.T) {
 	factory := NewFactory()
 	defaultGRPCSettings := &configgrpc.ServerConfig{
 		NetAddr: confignet.AddrConfig{
-			Endpoint:  testutil.GetAvailableLocalAddress(t),
+			Endpoint:  "127.0.0.1:0",
```

Comment:
I believe this is already what `GetAvailableLocalAddress` no? https://github.com/open-telemetry/opentelemetry-collector/blob/9ef6356835b5dfb784f47f4455854136af3bac85/internal/testutil/testutil.go#L115

Reply:
this is safe once, but the same config is reused throughout, and there is a slight chance the port might be taken.